index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>ACME Inc. Pet Food</title>
+</head>
+<body>
+  <div id="products">
+    <table>
+      <caption>Dog Food</caption>
+      <thead>
+        <tr>
+          <th>Brand:</th>
+          <th>Types/Prices:</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td class="brand-dog"></td>
+          <td class="types-dog"></td>
+        </tr>
+        <tr>
+          <td class="brand-dog"></td>
+          <td class="types-dog"></td>
+        </tr>
+      </tbody>
```

Comment:
This is not great practice.  While it's awesome how you're targeting the individual classes and inserting the data into the DOM in your JavaScript, doing it like this makes your HTML and data much too tightly coupled.

What if I wanted to add another brand of dog food to my JSON?  You'd have to then go back and add another ```<tr>``` and two ```<td>```s to the HTML, repeat for any changes.  The point of pulling the data out into the JSON is so that you can add/remove/modify _just_ the JSON and have the page do its thing with no tweaking.

better practice would be to slap an ID on that ```<tbody>``` tag, and then build the ```<tr>``` and ```<td>``` tags inside your loops in the JavaScript, that way, new items could be added to the JSON and they'd create their own elements in the page without having to edit it by hand.